libsolidity/formal/PredicateInstance.cpp

```diff
@@ -48,7 +50,8 @@ smtutil::Expression nondetInterface(
 	unsigned _postIdx)
 {
 	auto const& state = _context.state();
-	std::vector<smtutil::Expression> stateExprs{state.errorFlag().currentValue(), state.thisAddress(), state.abi(), state.crypto()};
+	std::vector<smtutil::Expression> stateExprs{state.errorFlag().currentValue(), state.thisAddress(),
+		state.abi(), state.bytesConcat(0), state.crypto()};
```

Comment:
Why `bytesConcat` is called with an argument while `abi` and `crypto` are without argument? In other cases, `bytesConcat` follows  what these two are doing, no?